packages/lib/utils/ActionLogger.ts

```diff
@@ -41,4 +45,12 @@ export default class ActionLogger {
 
 		return source;
 	}
+
+
+	// Disabling the action logger globally can be useful on Joplin Server/Cloud
+	// when many deletions are expected (e.g. for email-to-note).
+	private static disabled = false;
+	public static setDisabled(disabled: boolean) {
+		this.disabled = disabled;
+	}
```

Comment:
Sorry could we actually have the same interface as the Logger class? So an `enabled` property that would be `true` by default. This is to avoid double-negatives which are generally more difficult to reason with.